docs: update uninstall to include Managed updates v2 (#53066)
gravitational/teleport: docs/pages/admin-guides/management/admin/uninstall-teleport.mdx

  $ sudo rm -f /usr/local/bin/teleport
  $ sudo rm -f /usr/local/bin/tsh
  $ sudo rm -f /usr/local/bin/fdpass-teleport
  $ sudo rm -f /usr/local/bin/teleport-update
  ```

  </TabItem>
  <TabItem label="Managed Updates v2">

>   The `teleport-update uninstall --force` will remove all Teleport binaries regardless if there is a previous repository install to fall back to.

Reviewer: The paragraph introducing the `Tabs` component tells the reader to choose a tab based on their Linux distribution, but this tab isn't a Linux distribution. I would make this a separate H3 before Linux. I would also briefly explain how to tell if the reader is using Managed Updates v2, since this is a feature that we are rolling out.

Reviewer: @stevenGravy Just following up on this comment and whether there's anything else to do here. Thanks!

>   Note that this will not execute the uninstall if the previous step of disabling a `teleport` systemd is not completed.

Reviewer 2: ```suggestion
  Note that this will not execute the uninstall if Teleport is currently running.
```
We relax this restriction in the next release.
